### PR TITLE
Update scoreboard and game stage background.  Add borders and shading…

### PR DESCRIPTION
**Pull request by kaplanmichael**
… to enhance readability.

### CODE REVIEW

File: src/components/scoreboard/scoreboard.css [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-    top: 0px;
-    width: 7vw;
+    right: 10px;
+    top: 5px;
+}
+
+.component-scoreboard.frame {
+    border: 30px solid pink;
+    height: auto;
+    border-image: url("//s3-us-west-2.amazonaws.com/s.cdpn.io/4273/frame.png") 100;
```
Comment (MichaelFoss) [inline src/components/scoreboard/scoreboard.css:16]: Point this to the image in the repo, not on Amazon.

File: src/App.js [hunk 2]
```diff
@@ -154,7 +155,7 @@ class App extends Component {
     render() {
         return (
             <Loop>
-                <Stage style={ { background: '#000' } }>
+                <Stage style={ { backgroundColor: '#000 ', backgroundImage: 'url(' + theme1Bkgd + ')',  backgroundSize: '180px 180px'  } }>
```
Comment (MichaelFoss) [inline src/App.js:158]: We should probably export this to a stylesheet if it gets any lengthier.

**Review by MichaelFoss — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 2 above)
Comment (kaplanmichael) [reply src/App.js:158]: Agreed.  In a future iteration, I see us implementing Themes such that we could change the look (tile, apple color, scoreboard font color) of the game board if, say, all apples are eaten or something.  That would require changing the behaviour of the game-kit stuff were using so I didn't quite have the time to do that yet.

File: src/components/scoreboard/scoreboard.css (hunk 1 above)
Comment (MichaelFoss) [reply src/components/scoreboard/scoreboard.css:16]: Will allow this to go in for now, but will open issue to fix it.

**Review by MichaelFoss — APPROVED** [review]

**Pull request merged**